static/js/index.js

```diff
@@ -2067,11 +2098,20 @@ function loadData () {
                         if (oldPokemon.updated !== pokemon.updated) {
                             oldPokemon.updated = pokemon.updated;
                         }
-
+                        // TODO: Check if IV = 100%
+                        if (oldPokemon.expire_timestamp >= ts && !oldPokemon.pokemonTimerSet && oldPokemon.iv === 100) {
```

Comment:
```suggestion
                        if (oldPokemon.expire_timestamp >= ts && !oldPokemon.pokemonTimerSet && oldPokemon.iv === 100) {
```
Maybe set this value in config instead of hard coded?

Reply:
Oops the property `.iv` actually doesn't exist, I need to delete that. I'll add the minimum IV value to the config though.